views/entry/edit-basic.php

```diff
@@ -55,7 +55,9 @@
     </div>
 
     <div class="row">
-        <div class="col-md-6"></div>
+        <div class="col-md-6">
+            <?= $form->field($calendarEntryForm->entry, 'location')->textInput(['maxlength' => 64]) ?>
```

Comment:
Here you use the entry (which is good) so we do not need the location property in the form model.